modules/search/class.jetpack-search.php

```diff
@@ -1610,7 +1610,17 @@ public function get_filters( WP_Query $query = null ) {
 			} // End foreach().
 		} // End foreach().
 
-		return $aggregation_data;
+		/**
+		 * Modify the filters returned by get_filters()
```

Comment:
Could you expand on that title a bit so we know what filters this is referring to?    That information will end up being displayed in [our codex](https://developer.jetpack.com/module/search/), so it's important to be as detailed as possible for anyone looking for that filter in the future.

Reply:
Added some more info, let me know if that's ok. 👍